### DISCUSSION TITLE
This. Is. Cool.

### DISCUSSION THREAD

**Discussion by TechStudent10** [KendallDoesCoding/KendallDoesCoding#9] [General]
![image](https://user-images.githubusercontent.com/76978184/154864311-7cc3204d-5eee-4060-bafe-44ec75ee4248.png)

**Comment by KendallDoesCoding** [comment]
Yes. But it shows there for people you follow and people you've seen repos
of before. I think even your repo came there once or twice, maybe not I
dont remember lol

**Comment by KendallDoesCoding** [comment]
@TechStudent11 Have a look at this, the stuff that I didn't put a red line over is how it recommended stuff to me.

![image](https://user-images.githubusercontent.com/84712013/154929041-d67306c9-5ef3-4d15-846d-02da7d0334b6.png)

For you, the reason would've been, contributing the repos from me, even for "portfolio" you made a PR.

**Reply by TechStudent10** [reply]
1. Why are you using light mode?
2. My explore is always filled with Discord Bots. It's 1000/10 annoying

**Reply by KendallDoesCoding** [reply]
- I tried darkmode, but that doesn't really suit me got Github specifically, but I use darkmode for mostly all websites other than GitHub.. Github profile icons get messed up sometimes, and when I was editting code on GitHub, It just wasn't suiting my eyes for some reason.. Now, I use git bash though via VS Code so I may shift to darkmode
- There should be a way to be like "I don't like this suggestion", like youtube has for the recommendation tab, if it doesn't already exist.

**Reply by KendallDoesCoding** [reply]
But yeah, also the reason is you star lots of "Discord Bots" lol

**Reply by TechStudent10** [reply]
Yeah that makes sense. BUT STILL it's annoying. The reason I starred a lot of them was because they were cool. Now I don't star Discord Bots because I want to get rid of them from the explore.

Don't get me wrong, I have a lot of respect for people who's jobs are to make bots, but I don't want them being the only thing that's popping up in explore.

**Reply by KendallDoesCoding** [reply]
Yeah, makes sense... Discord bots are pretty cool ngl and there's no problem starting them, just GitHub should implement this feature lol.

> There should be a way to be like "I don't like this suggestion", like youtube has for the recommendation tab, if it doesn't already exist.

**Reply by KendallDoesCoding** [reply]
Yeah, makes sense... Discord bots are pretty cool ngl and there's no problem starting them, just GitHub should implement this feature lol.

> There should be a way to be like "I don't like this suggestion", like youtube has for the recommendation tab, if it doesn't already exist.

**Reply by KendallDoesCoding** [reply]
Yeah, makes sense... Discord bots are pretty cool ngl and there's no problem starting them, just GitHub should implement this feature lol.

> There should be a way to be like "I don't like this suggestion", like youtube has for the recommendation tab, if it doesn't already exist.

**Reply by TechStudent10** [reply]
Why did you spam 3 times?

**Reply by KendallDoesCoding** [reply]
Oh god sorry, internet wasn't working so I pressed the "Reply" button many times, and it went 3 times for some reason.

**Reply by KendallDoesCoding** [reply]
> There should be a way to be like "I don't like this suggestion", like youtube has for the recommendation tab, if it doesn't already exist.

@TechStudent11 I put this suggestion in the github feedback repo: https://github.com/github/feedback/discussions/12605
